pyproject.toml

```diff
@@ -13,12 +13,12 @@ minversion = "6.0"
 log_cli_level = "INFO"
 asyncio_mode = "auto"
 markers = [
-    "ha_tests",
-    "database_tests",
-    "db_router_tests ",
-    "shared_db_tests",
-    "mysql_interface_tests",
-    "healing_tests",
+    "ha_tests: high-availability tests",
+    "database_tests: database interface relation tests",
+    "db_router_tests: db-router legacy interface relation tests",
+    "shared_db_tests: shared_db legacy interface relation tests",
+    "mysql_interface_tests: mysql legacy interface relation tests",
+    "healing_tests: torture tests",
```

Comment:
Not sure this is the best terminology - perhaps rename this "stress tests"?

Reply:
ops - like your suggestion